api/src/users/controllers/users-controller.ts

```diff
@@ -15,29 +15,32 @@ import {
 import { UsersService } from "users/services/users.service";
 import { JwtAuthGuard } from "auth/guards/jwt-auth.guard";
 import { UpdateUserDto } from "users/model/dto/update-user.dto";
-import { UsersAvatarService } from "usersAvatar/services/usersAvatar.service";
 import { FileInterceptor } from "@nestjs/platform-express";
 import { diskStorage } from "multer";
 import { DeleteFileOnErrorFilter } from "filters/delete-file-on-error-filter";
 import {
     ApiBadRequestResponse,
     ApiBearerAuth,
+    ApiConflictResponse,
     ApiConsumes,
+    ApiForbiddenResponse,
     ApiInternalServerErrorResponse,
     ApiNotFoundResponse,
     ApiOkResponse,
     ApiTags,
     ApiUnauthorizedResponse
 } from "@nestjs/swagger";
 import IUploadedFile from "users/types/uploaded-file.interface";
+import { UserReq } from "users/decorators/user.decorator";
+import { UsersAvatarService } from "usersAvatar/services/usersAvatar.service";
 
 @ApiBearerAuth("defaultBearerAuth")
 @ApiTags("users")
 @Controller("users")
 class UsersController {
     constructor(
         private readonly userService: UsersService,
-        private readonly usersAvatarService: UsersAvatarService
+        private readonly userAvatarService: UsersAvatarService
```

Comment:
This is very inconsistent. Look other controllers. For me it should be `usersService` and `usersAvatarService`. But it's up to You. If You want to to correct naming choose one and fix in all project occurrences.

Reply:
Ye, you're right i will fix it

Reply:
Done